diskplorer.py

```diff
@@ -205,7 +215,29 @@ def global_section():
             job_names = generate_job_names(f'job(r_idx={read_iops_step},w_idx={write_bw_step},write_bw={write_bw},r_iops={read_iops})')
             read_iops = max(read_iops, 10)   # no point in a write-only test
             nr_cpus = args.cpus
+            read_offset_str = ''
+
             if write_bw > 0:
+                if args.run_sequential_trim:
+                    out(textwrap.dedent(f'''\
+                        [prepare_data_for_trim(r_idx={read_iops_step},w_idx={write_bw_step},write_bw={write_bw},r_iops={read_iops})]
+                        '''))
```

Comment:
This preparation step can be run at maximum bandwidth, no?

Can we share the data prepared by write with the trim? This doesn't work if we have to adjust the read range, but perhaps we can pre-calculate it.